src/20241021_Multi_GHOCCIP150Upgrade/GHOCCIP150Upgrade.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "GHO CCIP 1.50 Upgrade"
+author: "Aave Labs"
+discussions: "https://governance.aave.com/t/bgd-technical-maintenance-proposals/15274/51"
+---
+
+## Simple Summary
+
+## Motivation
+
+## Specification
+
+## References
+
+- Implementation: [AaveV3Ethereum](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20241021_Multi_GHOCCIP150Upgrade/AaveV3Ethereum_GHOCCIP150Upgrade_20241021.sol), [AaveV3Arbitrum](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20241021_Multi_GHOCCIP150Upgrade/AaveV3Arbitrum_GHOCCIP150Upgrade_20241021.sol)
+- Tests: [AaveV3Ethereum](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20241021_Multi_GHOCCIP150Upgrade/AaveV3Ethereum_GHOCCIP150Upgrade_20241021.t.sol), [AaveV3Arbitrum](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20241021_Multi_GHOCCIP150Upgrade/AaveV3Arbitrum_GHOCCIP150Upgrade_20241021.t.sol)
+- Snapshot - Direct-to-AIP
+- [Discussion](https://governance.aave.com/t/bgd-technical-maintenance-proposals/15274/51)
+
```

Comment:
I think also better to reference this repo: https://github.com/aave/ccip

Reply:
Along with the commit hash to know where exactly the contracts were deployed

Reply:
agreed - [445d0f7](https://github.com/bgd-labs/aave-proposals-v3/pull/512/commits/445d0f7db3b54d71d922bd3e7897fb8193fd4fda) is this sufficient?